app/src/js/utils/table-config/granules.js

```diff
@@ -67,6 +67,16 @@ export const tableColumns = [
     // eslint-disable-next-line react/prop-types
     Cell: ({ cell: { value } }) => <CopyCellPopover cellContent={collectionLink(value)} id={`collectionId-${value}-popover`} popoverContent={collectionLink(value)} value={value} />,
   },
+  {
+    Header: 'Executions List',
+    accessor: 'granuleId',
+    Cell: ({ row: { original: { collectionId, granuleId } } }) => (// eslint-disable-line react/prop-types
+      <Link to={(location) => ({ pathname: `/executions/executions-list/${encodeURIComponent(collectionId)}/${encodeURIComponent(path.basename(granuleId))}` })}>link</Link>
+    ),
+    disableSortBy: true,
+    width: 90,
+    id: 'execution-list'
+  },
```

Comment:
Only real thing I'm noticing is that we have this column and we also still have the 'Execution' column. Should we change that to 'Latest Execution' maybe? Or get rid of it entirely?

Reply:
I suggest to get rid of it entirely.

Reply:
Fair question. I'll get it removed.

Reply:
@dopeters and @jjmccoy: The Execution column has been removed. I also adjusted the granules integration tests to accommodate that change.